hepcrawl/spiders/elsevier_spider.py

```diff
@@ -797,18 +812,20 @@ def parse_node(self, response, node):
 
         if len(keys_missing) > 0:
             sd_url = self._get_sd_url(xml_file)
-            request = Request(sd_url, callback=self.scrape_sciencedirect)
-            request.meta["info"] = info
-            request.meta["keys_missing"] = keys_missing
-            request.meta["node"] = node
-            request.meta["xml_url"] = xml_file
-            request.meta["handle_httpstatus_list"] = self.ERROR_CODES
-            return request
+            try:
+                request = Request(sd_url, callback=self.scrape_sciencedirect)
+                request.meta["info"] = info
+                request.meta["keys_missing"] = keys_missing
+                request.meta["node"] = node
+                request.meta["xml_url"] = xml_file
+                request.meta["handle_httpstatus_list"] = self.ERROR_CODES
+                return request
+            except ValueError:
```

Comment:
Can you add a comment here explaining the reason for catching this potential exception?